xlint

```diff
@@ -400,7 +400,7 @@ for argument; do
 	scan 'short_desc=["'\''][a-z]' "short_desc should start uppercase"
 	scan 'short_desc=["'\''](An?|The) ' "short_desc should not start with an article"
 	scan 'short_desc=["'\''][\t ]' "short_desc should not start with whitespace"
-	scan 'short_desc=["'\''].{73}' "short_desc should be less than 72 chars"
+	scan 'short_desc=["'\''].{73}' "short_desc should be at most 72 chars"
```

Comment:
Should be 74 to actually enforce "at most 72".

Reply:
I actually confirmed that the existing check is correct after you left the IRC channel.

Reply:
what's the output of `xlint qt6-qt5compat`?

Reply:
Never mind, I was wrong there; I was using `echo foo | wc` to count characters, but the count was inflated by one because of the trailing `\n` output by `echo`, hence why I thought the existing behavior was correct. I'll push your suggested fix.